src/templates/summit-landing-page.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { graphql } from 'gatsby';
+import { connect } from 'react-redux'
+import { HTMLContent } from '../components/Content'
+import Layout from '../components/Layout'
+import TopBar from "../components/TopBar";
+import NavbarV2 from "../components/NavbarV2";
+import SEO from "../components/SEO";
+import HeaderImage from "../components/HeaderImage";
+import hero from '../../static/img/summit-landing/summit-landing-hero.png';
+import SponsorBanner from "../components/SponsorBanner";
+import SubHeader from "../components/SubHeader";
+import PreviousSummits from "../components/PreviousSummits";
+import SummitCard from "../components/SummitCard";
+
+import summitCardAsia from '../../static/img/summit-landing/cards/summit-asia.png';
+import summitCardEurope from '../../static/img/summit-landing/cards/summit-europe25.png';
+import CommunityEvents from "../components/CommunityEvents";
+
+export const SummitLandingPageTemplate = ({
+                                       isLoggedUser,
+                                       title,
+                                       subTitle,
+                                       content,
+                                       contentComponent
+                                     }) => {
+  return (
+    <div>
+      <div className="wrapper project-background">
+        <TopBar />
+        <NavbarV2 isLoggedUser={isLoggedUser} />
+        <HeaderImage backgroundImage={hero} />
+        <SubHeader />
+        <SummitCard background={summitCardAsia} date={'September 2 & 3, 2024'} location={'Suwon Convention Center, Suwon, South Korea'} notification={true}/>
```

Comment:
@tomrndom could we use a js array ? like here
https://github.com/OpenStackweb/osf-website/blob/a0840aadf8de4bc3f9c3d1bb5da161859f944215/src/components/PreviousSummits/index.jsx#L9